tests/sentry/api/endpoints/test_project_repo_path_parsing.py

```diff
@@ -144,6 +144,34 @@ def test_member_can_access(self):
         resp = self.make_post(source_url, stack_path, user=member)
         assert resp.status_code == 200, resp.content
 
+    def test_backslash_short_path(self):
```

Comment:
Good tests well done.

Please also add a few unit test cases for `PathMappingSerializer`.